src/obs_interface.cpp

```diff
@@ -1004,12 +999,12 @@ void ObsInterface::getSourcePos(std::string name, vec2* pos, vec2* size, vec2* s
   obs_sceneitem_t *item = obs_scene_find_source(scene, name.c_str());
 
   if (!src) {
-    blog(LOG_WARNING, "Did not find source for video source: %s", name);
+    blog(LOG_WARNING, "Did not find source for video source: %s", name.c_str());
```

Comment:
Full warning here:
```
Cannot pass object of non-trivial type 'std::string' (aka 'basic_string<char, char_traits<char>, allocator<char>>') through variadic function; call will abort at runtime
```

This is a spot where clangd is stricter than gcc/msc; no behavior change here